README.md

```diff
@@ -1,67 +1,36 @@
-* [Testing](configuration/gateway-testing.md)
-* [Recommended phones](configuration/gateway-phones.md)
-
-### Standard
-
-* [Installing a standard project](configuration/installing-a-standard-project.md)
-
-## Configuration
-
-* [Forms](configuration/forms.md)
-* [App settings](configuration/app-settings.md)
-* [Replication](configuration/replication.md)
-* [Tasks](configuration/tasks.md)
-* [Targets](configuration/targets.md)
-* [Contact summary cards](configuration/contact-summary.md)
-* [Sentinel transitions](configuration/transitions.md)
-* [CouchDB authentication](configuration/couchdb-authentication.md)
-* [Garden](configuration/garden.md)
-* [Replicating with Futon](configuration/replicate.md)
-* [Variables in messages](configuration/variables-in-messages.md)
-* [Configuring patient shortcode length](configuration/shortcode-identifiers.md)
-* [Accessing CouchDB directly](configuration/direct-access.md)
-* [Install SSL Cerificates to VM](configuration/install-ssl-certificates-to-vm.md)
-* [Connect to RDBMS on windows](configuration/connecting-to-rdbms-in-windows.md)
-
-## Support and Troubleshooting
-
-* [Dealing with conflicts](troubleshooting/conflicts.md)
-* [Troubleshooting - Quick Pointers and Tips](troubleshooting/troubleshooting-quick-pointers.md)
-* [Replicating production data locally](troubleshooting/replicating-production-locally.md)
-* [Obtaining production logs](troubleshooting/obtaining-logs.md)
-
-## Development
-
-### Code
-
-* [Architecture](development/architecture.md)
-* [DB schema](development/db-schema.md)
+### How we work
 * [Developer environment setup](https://github.com/medic/medic/blob/master/README.md)
+* [Using Git](development/using-git.md)
+* [Using npm](development/using-npm.md)
+* [Our workflow](development/workflow.md)
+* [Culture](development/culture.md)
 * [Style guide](development/style-guide.md)
 * [Updating translations](development/translations.md)
-* [Our workflow](development/workflow.md)
 * [Releasing our app](development/releasing.md)
-* [Virtual appliance build environment](development/mmva-build.md)
+
+### Technical overviews
+* [Architecture](development/architecture.md)
+* [DB schema](development/db-schema.md)
+* [Message states](user/message-states.md)
 * [How to migrate dashboard to SSL](development/update-markets.md)
-* [Using Git](development/using-git.md)
-* [Using npm](development/using-npm.md)
 
-### Culture
+## Building Community Health Applications
+An introduction and reference for developing community health applications is provided in [Developing Community Health Applications](https://github.com/medic/medic-docs/blob/master/configuration/developing-community-health-applications.md). This overview covers building forms, tasks, targets, and contact profiles.
 
-* [Culture](development/culture.md)
+Additional topics for building apps using the framework are available [here](https://github.com/medic/medic-docs/blob/master/configuration/README.md).
```

Comment:
Using the word "here" as the link text is bad form: https://medium.com/@heyoka/dont-use-click-here-f32f445d1021

How about:

"Read more about [building apps using the framework](https://github.com/medic/medic-docs/blob/master/configuration/README.md)."